specification/communication/Communication.Messages/models.tsp

```diff
@@ -450,6 +476,315 @@ model NotificationMessagesSendRequestBody {
+@doc("The options of the outbound delivery strategy for messages sent by emplyees in a conversation.")
+union OutboundDeliveryStrategyKind {
+  string,
+
+  @doc("Delivery strategy for internal only.")
+  internalOnly: "internalOnly",
+
+  @doc("Delivery strategy for all participants.")
+  allParticipants: "allParticipants",
+}
+
+@clientName("ConversationMessage", "csharp")
+@added(Versions.c2025_03_01_Preview)
+@doc("Details of a message.")
+model Message {
+  @doc("Content of the message.")
+  content: string;
+}
+
+@added(Versions.c2025_03_01_Preview)
+@discriminator("kind")
+@resource("participants") // participants add/remove
+@parentResource(Conversation)
+@doc("Advanced Messaging conversation participant.")
+model Participant {
+  @key
+  @visibility("read")
+  @doc("Participant Identifier.")
+  id: string; // 8:acs:xxx
+
+  @doc("Participant display name.")
+  displayName?: string;
+
+  @doc("The type discriminator describing a participant type.")
+  kind: ParticipantKind;
+}
+
+@added(Versions.c2025_03_01_Preview)
+@doc("Internal conversation participant.")
+model InternalParticipant extends Participant {
+  @doc("Participant type is internal.")
+  kind: ParticipantKind.internal;
+
+  @doc("The internal platform identifiers for the participant.")
+  contact: Contact;
+}
+
+@added(Versions.c2025_03_01_Preview)
+@doc("External conversation participant.")
+model ExternalParticipant extends Participant {
+  @doc("Participant type is external.")
+  kind: ParticipantKind.external;
+
+  @doc("List of external platform identifiers for the participant.")
+  contacts: Contact[];
+}
+
+@added(Versions.c2025_03_01_Preview)
+@doc("The type of a participant in a threaded conversation.")
+union ParticipantKind {
+  string,
+
+  @doc("A participant internal to Azure Communication Services.")
+  internal: "internal",
+
+  @doc("A participant on other platforms")
+  external: "External",
+}
+
+@added(Versions.c2025_03_01_Preview)
+@doc("Request payload for sending a conversation message.")
+model SendConversationMessageRequest {
+  @doc("Details of a send conversation message request.")
+  request: ConversationMessageContent;
+
+  @doc("The options of the outbound delivery strategy for messages sent by emplyees in a conversation.\r\nSupports internalOnly, allChannels.")
+  outboundDeliveryStrategy?: OutboundDeliveryStrategyKind;
+}
+
+@added(Versions.c2025_03_01_Preview)
+@doc("Result of the send conversation message operation.")
+model SendConversationMessageResult {
+  @doc("A server-generated Advanced Messaging conversation message id.")
+  messageId: string;
+}
+
+@added(Versions.c2025_03_01_Preview)
+@resource("messages/conversations")
+@doc("A conversation.")
+model Conversation {
+  @key
+  @visibility("read") // this model is only used as a parameter
+  @doc("The conversation ID.")
+  conversationId: string;
+
+  @doc("The conversation topic.")
+  topic?: string;
+
+  @doc("List of delivery channel IDs.")
+  deliveryChannelIds?: string[];
```

Comment:
Is it possible for the business to own multiple Whatsapp/SMS/Email accounts and need to communicate with different external participants over different business accounts?